in_toto/model.go

```diff
@@ -15,6 +15,8 @@ import (
 	"strings"
 	"time"
 
+	latest "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.1"
```

Comment:
To be honest, I am a little bit uncomfortable with "latest". This is a very generic key word. Can we rename this slsa?

Reply:
What do you think @adityasaky ? Or is it just me?

Reply:
Yeah, I think we can get more specific than `latest` to avoid any confusion / reuse in local blocks.

Reply:
sgtm, renamed to `slsa` :)